### ISSUE TITLE
tanfolyam naptár

### ISSUE THREAD

**Issue by vacsi** [rekavarga/first-steps#7]
Lesz majd vmi tanfolyam naptár is? Hogy mikor milyen tanfolyamok indulnak? Ezt vhol úgy kéne felrakni, hogy még a jelentkezem gomb előtt legyenek. Akár csak felsorolás szerűen.
pl.: 
Következő induló tanfolyamaink:
Kölyökkutya tanfolyam: június 18
Felnőtt tanfolyam: június 25

**Comment by buzag** [comment]
Igen, ez tervben van. En azt javasoltam rekanak, hogy facebook eventek legyenek az idopontok, mert ahhoz talaljuk ki a workflow-t: kepek feltoltese, tag-eles, hozzatartozo beszelgetesek.
